lib/namespace.js

```diff
@@ -25,10 +25,15 @@ var Namespace = createClass({
   },
 
   /*
-   * Use a namespace plugin.
+   * Use a namespace plugin or load a generic plugin.
    */
   use: function(plugin) {
-    plugin.namespace({base: this});
+    if (plugin.namespace) {
+      plugin.namespace({base: this});
+    }
+    else if (plugin.load) {
```

Comment:
I think this makes sense. I think if someone used both but only saw `namespace` used, it would be confusing.